source/winVersion.py

```diff
@@ -28,38 +126,28 @@ def isUwpOcrAvailable():
 
 
 WIN10_VERSIONS_TO_BUILDS = {

```

Comment:
As I understand these strings are actually 'release names'. If others agree, it would be good if we could rename this.
```suggestion
WIN10_RELEASE_NAME_TO_BUILDS = {
```